README.md

```diff
@@ -1,44 +1,58 @@
 
-Inspired by [GTFS](https://developers.google.com/transit/gtfs/reference/) and [GBFS](https://github.com/NABSA/gbfs). Specifically, the goals of the Mobility Data Specification (**MDS**) are to provide API and data standards for municipalities to help ingest, compare and analyze *mobility as a service* provider data. 
+The Mobility Data Specification (MDS) helps the City ingest and analyze information from for-profit companies who operate dockless scooters and bicycles in the public right-of-way. MDS is a key piece of digital infrastructure that helps cities and regulators such as Los Angeles Department of Transportation (LADOT) understand how dockless mobility operate. For example, this data helps LADOT carry out its core responsibilities of safety, equity, and congestion relief.
 
-The specification is a way to implement realtime data sharing, measurement and regulation for municipalities and *mobility as a service* providers. It is meant to ensure that governments have the ability to enforce, evaluate and manage providers. 
+Mobility providers are required to share data with LADOT as part of the City of Los Angeles' Dockless Mobility Permit. Standardizing data collection between different providers improves cooperation, innovation, and efficiency of the City's transportation network.
 
 **MDS** is currently comprised of two distinct components:
 
-* The [`provider`][provider] API is to be implemented by *mobility as a service* providers, for data exchange and operational information that a municipality will query. `provider` presents the historical view of operations.
+* The [`provider`][provider] Application Program Interface (API) was created in May 2018 to be implemented by mobility providers. When a municipality queries information from a mobility provider, the Provider API automatically generates a historical view of operations in a standard format.
 
-* The [`agency`][agency] API is to be implemented by *municipalities* and other regulatory agencies, for providers to query and integrate with during operations. `agency` provides tools to inform and permit future operations.
+* The [`agency`][agency] API was created in April 2019 to be implemented by regulatory agencies. Providers query the Agency API when an event occurs, like a trip starting or ending.
 
-Cities and regulators can choose best how to implement *Agency* and *Provider* either separately, concurrently, or by endpoint. 
+Cities and regulators can choose best how to implement *Agency* and *Provider* either separately, concurrently, or by endpoint.
 
-## Versions 
+## Use Cases
```

Comment:
This whole section feels out of place in the main README document for the spec, or at least this high up in the document. IMHO the primary audience of this document are technical folks looking to implement or utilize some aspect of MDS. The *Versions* section that was here before is probably more relevant to a wider audience.

Reply:
moved down lower in the spec in a forthcoming PR.